cobra/mit/access.py

```diff
@@ -125,12 +126,15 @@ def lookupByClass(self, classNames, parentDn=None, propFilter=None):
             dnQuery = DnQuery(parentDn)
             dnQuery.classFilter = classNames
             dnQuery.queryTarget = 'subtree'
-            if propFilter:
-                dnQuery.propFilter = propFilter
+            self.__setQueryParams(dnQuery, kwargs)
             mos = self.query(dnQuery)
         else:
             classQuery = ClassQuery(classNames)
-            if propFilter:
-                classQuery.propFilter = propFilter
+            self.__setQueryParams(classQuery, kwargs)
             mos = self.query(classQuery)
         return mos
+
+        def __setQueryParams(self, query, queryParams):
```

Comment:
Please change setQueryParams to be a staticmethod and remove self from the arguments.